hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/jobhistory/TestHistoryViewerPrinter.java

```diff
@@ -160,6 +162,10 @@ public void testHumanPrinter() throws Exception {
         LINE_SEPARATOR, outStr);
   }
 
+  private void compareStrings(String str1, String str2) {
+    Assertions.assertThat(Arrays.asList(str1.trim().split("\n"))).containsExactlyInAnyOrderElementsOf(Arrays.asList(str2.trim().split("\n")));
```

Comment:
* This could be made a little more readable by extracting the two chunks.
* Make the strings final.
* Name the method something like assertEqualLines().
* Make method static.